app/uk/gov/hmrc/gform/views/form/form.scala.html

```diff
@@ -68,7 +68,7 @@
 @govukButton = @{new govukButton()}
 
 @main_template(
-  title = heading,
+  title = if(isDeclaration) heading else "",
```

Comment:
I don't think we want declaration to be treat differently, since declaration title is also `SmartString` so it can contains PII.  So `title` in here should be always empty `""`